lib/user_space/timer.c

```diff
@@ -67,8 +67,8 @@ M0_TL_DEFINE(tid, static, struct m0_timer_tid);
    gettid(2) implementation.
    Thread-safe, async-signal-safe.
  */
-static pid_t gettid() {
-
+static pid_t m0_gettid()
```

Comment:
What function name is this gettid() conflicting with?
The system call gettid()?

Usually m0_ prefix is used for M0_INTERNAL or M0_EXTERNAL.
For static function, we don't use the m0_ prefix.

Can we change it to _gettid()?

Reply:
Done.

Reply:
And yes, it conflicts with glibc version of gettid().